Makefile

```diff
@@ -207,6 +207,8 @@ TEST_PACKAGES = \
 .PHONY: tinygo-test
 tinygo-test:
 	$(TINYGO) test $(TEST_PACKAGES)
+	# until "test testing" passes
+	cd tests/tinygotest && $(TINYGO) test benchmark_test.go
```

Comment:
I wonder why this doesn't work. Ideally I'd like to see this fixed first but maybe it's better to merge this PR first so it doesn't stall any longer.

Reply:
Filed https://github.com/tinygo-org/tinygo/issues/2027

Reply:
> maybe it's better to merge this PR first so it doesn't stall any longer.

Agreed.